### PR TITLE
Zkevm integration

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to related to #1617 #1615 #1618 #1619 #1614 #1620

## Summary by CodeRabbit

- **New Features**
  - Introduced new scan API keys for various networks to enhance service integrations.
  - Added support for BASE, BASE_SEPOLIA, ZKEVM_MAINNET, and ZKEVM_CARDONA networks.
  - Introduced a read-only replica for the local PostgreSQL database.
  - Added new database migrations to handle donation records and base chain tokens.
  - Enhanced network handling and error logging for BASE and ZKEVM networks.

- **Bug Fixes**
  - Adjusted donation and project filtering to support new networks in test cases.

- **Refactor**
  - Updated SQL query structures and field selection logic in several resolver classes.

- **Documentation**
  - Included new environment variables and configurations in example and test environment files.

- **Tests**
  - Expanded test cases to cover new networks and features.

- **Chores**
  - Updated `package.json` test scripts to include additional test files.

### CODE REVIEW

File: migration/data/seedTokens.ts [hunk 1]
```diff
@@ -1881,6 +1881,198 @@ const seedTokens: ITokenData[] = [
...
+  {
+    name: 'ChainLink Token',
+    symbol: 'LINK',
+    decimals: 18,
+    address: '0x4b16e4752711a7abec32799c976f3cefc0111f2b',
+    coingeckoId: 'chainlink',
+    isGivbackEligible: false,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
```
Comment (Meriem-BM) [inline migration/data/seedTokens.ts:2022]: I think these two shouldn't have the same address `0xa61beb4a3d02decb01039e378237032b351125b4`, right?

File: migration/data/seedTokens.ts [hunk 2]
```diff
@@ -1881,6 +1881,198 @@ const seedTokens: ITokenData[] = [
...
+    coingeckoId: 'aave',
+    isGivbackEligible: false,
+    networkId: NETWORK_IDS.ZKEVM_MAINNET,
+  },
+  {
+    name: 'Wrapped BTC',
+    symbol: 'WBTC',
+    decimals: 8,
+    address: '0xea034fb02eb1808c2cc3adbc15f447b93cbe08e1',
+    coingeckoId: '',
```
Comment (Meriem-BM) [inline migration/data/seedTokens.ts]: Here you forgot the id I guess "wrapped-bitcoin"

**Review by Meriem-BM — COMMENTED** [review]
I just have 2 comments, overall the code LGTM!

File: migration/data/seedTokens.ts (hunk 2 above)
Comment (mohammadranjbarz) [reply migration/data/seedTokens.ts]: Thanks I added that

File: migration/data/seedTokens.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply migration/data/seedTokens.ts:2022]: @Meriem-BM Nice catch, there were same in the issue, I will ask Moenick to give me the correct one

Comment (mohammadranjbarz) [reply migration/data/seedTokens.ts:2022]: Done.

**Pull request merged**